### PR TITLE
Release notes 2022.07

### LINKED ISSUE #7500: chore: release notes 2022.07
release notes 2022.07

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #7500

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,97 @@ id: Release notes
...
+  - Updated `appendTo` in demos ([#7417](https://github.com/patternfly/patternfly-react/pull/7417))
+- **Modal:**
+  - Fixed tooltip visibility on modal open ([#7411](https://github.com/patternfly/patternfly-react/pull/7411))
+- **Options menu:**
+  - Exposed `menuAppendTo` in prop docs ([#7435](https://github.com/patternfly/patternfly-react/pull/7435))
+- **Popper:**
+  - Added mouse events for popper content ([#7399](https://github.com/patternfly/patternfly-react/pull/7399))
+- **Search input:**
+  - Updated to use panel instead of card in composable advanced search demo ([#7431](https://github.com/patternfly/patternfly-react/pull/7431))
+  - Updated tp use text input group instead native HTML elements ([#7414](https://github.com/patternfly/patternfly-react/pull/7414))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:54]: Typo "to"
```suggestion
  - Updated to use text input group instead native HTML elements ([#7414](https://github.com/patternfly/patternfly-react/pull/7414))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,97 @@ id: Release notes
...
+- [@patternfly/react-topology@4.68.3](https://www.npmjs.com/package/@patternfly/react-topology/v/4.68.3)
+- [@patternfly/react-virtualized-extension@4.68.3](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.68.3)
+
+### Components
+- **Application launcher:**
+  - Exposed `menuAppendTo` in prop docs ([#7435](https://github.com/patternfly/patternfly-react/pull/7435))
+- **Banner:**
+  - Added screen reader text ([#7377](https://github.com/patternfly/patternfly-react/pull/7377))
+- **Chip:**
+  - Added custom maxWidth prop ([#7387](https://github.com/patternfly/patternfly-react/pull/7387))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:26]: Format prop name to match other props
```suggestion
  - Added custom `maxWidth` prop ([#7387](https://github.com/patternfly/patternfly-react/pull/7387))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Two nits

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**